qpylib/rest_qpylib.py

```diff
@@ -52,7 +52,7 @@ def _add_headers(headers, version=None):
         headers['Version'] = version
 
     if headers.get('Host') is None:
-        headers['Host'] = gethostbyname(gethostname())
+        headers['Host'] = gethostbyname('localhost')
```

Comment:
This isn't mentioned in the pull request description.

Reply:
Will asked me to change this line while I was making the changes